src/cljs/commiteth/update_address.cljs

```diff
@@ -3,14 +3,16 @@
             [commiteth.common :refer [input dropdown]]
             [reagent.core :as r]
             [reagent.crypt :as crypt]
+            [clojure.string :as str]
             [cljs-web3.eth :as web3-eth]))
 
 
 (defn update-address-page []
   (let [db (rf/subscribe [:db])
         user (rf/subscribe [:user])
         updating-address (rf/subscribe [:get-in [:updating-address]])
-        address (r/atom @(rf/subscribe [:get-in [:user :address]]))]
+        address (r/atom (some-> @(rf/subscribe [:get-in [:user :address]]) 
+                                str/lower-case))]
```

Comment:
Why is this conversion to lower case really needed? I'm not an expert in Ethereum addresses, but I think that capitalisation plays a role for some semantics. Maybe @3esmit or @ can help us.

Reply:
I ran into an issue with address string case. MetaMask account number contains mixed case symbols, for instance! E.g. this is one of my account numbers copied from MetaMask: `0x111d500fe567696D0224A7292D47AF11e8A4bCB4`.

So I think that this conversion is necessary when equality comparison is invoked. Of course we can force addresses into lowercase in the UI, and have a DB script to convert old possibly broken addresses that are already stored, but maybe this is a separate discussion.

Reply:
https://forum.ethereum.org/discussion/9220/eth-address-upper-and-lower-characters-does-not-matter

Just not sure what implications it has for the SOB right now.

Comparisons can probably be done in lowercase, but if possible it would be nice to save it to be able to perform that checksum in case we need it in the future.

Reply:
Changed to comparisons in lower-case. Curiously, addresses returned by web3 js object are all lowercase.